xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/test/java/org/xwiki/mentions/templates/MentionTest.java

```diff
@@ -74,24 +75,16 @@ void mention() throws Exception
     {
         TemplateManager templateManager = this.oldcore.getMocker().getInstance(TemplateManager.class);
         VelocityManager velocityManager = this.oldcore.getMocker().getInstance(VelocityManager.class);
-        VelocityContext velocityContext = velocityManager.getVelocityContext();
+        this.componentManager.registerComponent(ScriptService.class, "icon", this.iconManager);
+        this.componentManager.registerComponent(ScriptService.class, "date", this.dateScriptService);
+        this.componentManager.registerComponent(ScriptService.class, "localization", this.localizationScriptService);
```

Comment:
I don't understand this. It shouldn't be needed if you have the component in the `@ComponentList`.